cartridges/int_imgix_products_sfra/cartridge/models/product/productImages.js

```diff
@@ -10,36 +10,87 @@ const currentSite = require('dw/system/Site').getCurrent();
  * @param {Object} imageConfig - configuration object with image types
  */
 function Images(product, imageConfig) {
-    const imgixBaseURL = currentSite.getCustomPreferenceValue('imgixBaseURL') || '';
+  const imgixBaseURL =
+    currentSite.getCustomPreferenceValue("imgixBaseURL") || "";
+  const isBaseURLSet = imgixBaseURL.trim().length > 0;
+  const imgixDefaultParams =
+    currentSite.getCustomPreferenceValue("imgixProductDefaultParams") || "";
 
+  if (isBaseURLSet) {
+    // Here we transform the URL
     imageConfig.types.forEach(function (type) {
-        var images = product.getImages(type);
-        var result = {};
+      var images = product.getImages(type);
+      var result = {};
 
-        if (imageConfig.quantity === 'single') {
-            var firstImage = collections.first(images);
-            if (firstImage) {
-                result = [{
-                    alt: firstImage.alt,
-                    url: imgixBaseURL + firstImage.URL.toString(),
-                    title: firstImage.title,
-                    index: '0',
-                    absURL: firstImage.absURL.toString()
-                }];
-            }
-        } else {
-            result = collections.map(images, function (image, index) {
-                return {
-                    alt: image.alt,
-                    url: imgixBaseURL + image.URL.toString(),
-                    index: index.toString(),
-                    title: image.title,
-                    absURL: image.absURL.toString()
-                };
-            });
+      if (imageConfig.quantity === "single") {
+        var firstImage = collections.first(images);
+        if (firstImage) {
+          result = [
+            {
+              alt: firstImage.alt,
+              url:
+                imgixBaseURL +
+                firstImage.URL.toString() +
+                (isBaseURLSet && imgixDefaultParams
+                  ? `?${imgixDefaultParams}`
```

Comment:
Turns out SFCC doesn't like ES6 very much 
```suggestion
                  ? "?" + imgixDefaultParams
```